TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/claw/ClawConstants.java

```diff
@@ -0,0 +1,13 @@
+package org.firstinspires.ftc.teamcode.subsystems.claw;
+
+class ClawConstants {
+
+    protected static final String RIGHT_SERVO_ID = "rightGripper";
+    protected static final String LEFT_SERVO_ID = "leftGripper";
+    protected static final boolean FINGER_IS_OPEN_AT_START = true;
```

Comment:
Nitpicking: This is a litlle bit overkill, you don't have to create consts for boolean, passing true for `isOpen` is clear enought